base/shell/cmd/lang/cs-CZ.rc

```diff
@@ -531,7 +531,7 @@ title         new title\n"
     STRING_REPLACE_ERROR5 "Přístup odepřen - %s\n"
     STRING_REPLACE_ERROR6 "Nebyly nazeleny žádné soubory - %s\n"
     STRING_REPLACE_ERROR7 "Rozšířená chyba 32\n"
-    STRING_CMD_INFOLINE "  ReactOS Příkazový řádek                              Napsání HELP = Nápověda  "
+    STRING_CMD_COPYRIGHT "(C) Copyright 1998-%s ReactOS Team.\n\n"
```

Comment:
I do like your PR, but I would suggest to use the year 1996 here instead of 1998 (same as used in our shell32, and same as in our pictures for the session startup/shutdown.